.github/workflows/container_description.yml

```diff
@@ -5,7 +5,7 @@ on:
     paths:
       - "README.md"
       - ".github/workflows/container_description.yml"
-  branches: [ main, master ]
```

Comment:
oops. Maybe we can think about adding a linter for such workflows that don't run on each PR...
Maybe we could take advantage of this to adjust the name + some of the steps names under `PushQuayIoReadme` (they're still mentioning docker hub)

Reply:
Yea, probably a good TODO item. 

Fixed the step names.

Reply:
I'll create an issue.
A last one in line 35, but we can keep it as a "first contribution" to someone else ;)